Rocket.Chat/Extensions/UserDefaults+Group.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UserDefaults+Group.swift
+//  Rocket.Chat
+//
+//  Created by Matheus Cardoso on 3/1/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+extension UserDefaults {
+    static var group: UserDefaults {
+        // swiftlint:disable force_unwrapping
+        return UserDefaults(suiteName: AppGroup.identifier)!
```

Comment:
Do we need to force the unwrap here? I would say to try it or fail as a fatalError() and return the defaults.